bot/bot.py

```diff
@@ -180,6 +203,34 @@ def _remove_root_aliases(self, command: commands.Command) -> None:
         for alias in getattr(command, "root_aliases", ()):
             self.all_commands.pop(alias, None)
 
+    async def process_ext_not_found(self) -> None:
+        """
+        Process extensions which are in unloaded cache but not found on the bot.
+
+        Send a message to #dev-alerts with the extensions, and remove them from the cache,
+        """
+        await self.wait_until_guild_available()
+
+        if not self.unloaded_extensions:
+            return
+
+        cache_dict = await self.unloads_cache.to_dict()
+        extensions_msg = "\n- ".join(f"`{ext}`: {cache_dict.get(ext)}" for ext in self.unloaded_extensions)
```

Comment:
You should be able to get the message link for an extension straight from the rediscache without having to get the whole cache as a dictionary.